assets/css/_tab_bar.scss

```diff
@@ -53,9 +53,14 @@
   }
 }
 
-.m-tab-bar__refresh {
+.m-tab-bar__bottom-buttons {
   bottom: 0;
-  padding: 0.5rem;
+  display: flex;
+  flex-direction: column;
   position: absolute;
   width: 100%;
+
+  button {
+    padding: 0.5rem;
+  }
```

Comment:
Does this need `flex: 1 1 auto`?

Reply:
I don't think so, I'm not seeing any effect from that.